core/src/test/java/org/elasticsearch/search/suggest/SuggestBuilderTests.java

```diff
@@ -111,26 +224,37 @@ protected SuggestBuilder createMutation(SuggestBuilder original) throws IOExcept
         return mutation;
     }
 
-    public void testIllegalSuggestionName() {
-        try {
-            new SuggestBuilder().addSuggestion(null, PhraseSuggestionBuilderTests.randomPhraseSuggestionBuilder());
-            fail("exception expected");
-        } catch (NullPointerException e) {
-            assertEquals("every suggestion needs a name", e.getMessage());
+    @Override
+    protected SuggestBuilder readFrom(StreamInput in) throws IOException {
+        return SuggestBuilder.PROTOTYPE.readFrom(in);
+    }
+
+    public static SuggestBuilder randomSuggestBuilder() {
```

Comment:
This almost looks identical to createTestModel() in this test class, maybe you can make the later delegate to `randomSuggestBuilder()`.